artifacts/scripts/construct.sh

```diff
@@ -37,8 +37,8 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-if [ ! $# -eq 9 ]; then
-    echo "usage: $0 repo src_branch dst_branch dependent_k8s.io_repos kubernetes_remote subdirectory source_repo_org source_repo_name is_library"
+if [ ! $# -eq 10 ]; then
+    echo "usage: $0 repo src_branch dst_branch dependent_k8s.io_repos required_k8s.io_packages kubernetes_remote subdirectory source_repo_org source_repo_name is_library"
```

Comment:
is this k8s specific?

Reply:
no, not at all actually. Bad variable name. Will fix.

Reply:
Fixed